styles/deepl/catppuccin.user.css

```diff
@@ -259,10 +260,18 @@
     }
 
     // header
-    [class^="BasePageHeader-module--container"] {
+    .BasePageHeader-module--container--2By68 {
```

Comment:
Why change this? The previous version was more resistant to change as it still works even if the autogenerated bit at the end is different.

Reply:
the previous one didn't seem to work when I was testing.

Reply:
could probably just make it `[class*="BasePageHeader-module--"]`. it worked on my end before but ig they changed stuff